polkadot/runtime/common/src/impls.rs

```diff
@@ -67,29 +67,51 @@ where
 	}
 }
 
-pub fn era_payout(
-	total_staked: Balance,
-	total_stakable: Balance,
-	max_annual_inflation: Perquintill,
-	period_fraction: Perquintill,
-	auctioned_slots: u64,
-) -> (Balance, Balance) {
-	use pallet_staking_reward_fn::compute_inflation;
+/// Parameters passed into [`relay_era_payout`] function.
+pub struct EraPayoutParams {
+	/// Total staked amount.
+	pub total_staked: Balance,
+	/// Total stakable amount.
+	///
+	/// Usually, this is equal to the total issuance, except if a large part of the issuance is
+	/// locked in another sub-system.
+	pub total_stakable: Balance,
+	/// Ideal stake ratio, which is deducted by `legacy_auction_proportion` if not `None`.
+	pub ideal_stake: Perquintill,
+	/// Maximum inflation rate.
+	pub max_annual_inflation: Perquintill,
+	/// Minimum inflation rate.
+	pub min_annual_inflation: Perquintill,
+	/// Falloff used to calculate era payouts.
+	pub falloff: Perquintill,
+	/// Fraction of the era period used to calculate era payouts.
+	pub period_fraction: Perquintill,
+	/// Legacy auction proportion, which substracts from `ideal_stake` if not `None`.
+	pub legacy_auction_proportion: Option<Perquintill>,
+}
+
+/// A specialized function to compute the inflation of the staking system, tailored for polkadot
+/// relay chains, such as Polkadot, Kusama and Westend.
```

Comment:
```suggestion
/// A specialized function to compute the inflation of the staking system.
```

Nit but in this runtime it is implicit that the configs/impls are for Westend relay chain.